src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const fetchPeople = async () => (
+      setPeople(await getPeople())
+    );
```

Comment:
Will be better to create separate variable, store the result of getPeople function here, and then pass this variable to setPeople function.
Your code is short - but with separate variable it will be more understandable for other users